README.md

```diff
@@ -3,39 +3,83 @@
 ![Shakespeare](http://www.gomoxie.com/wp-content/uploads/Hamlet.jpg)
 >To be, or not to be...
 
-## Learning Objectives - The student should be able to..
+## Objectives
 
-* Explain that by making parts of their code *conditional*, they can execute different pieces of code based on certain conditions.
-* Explain that an *if* statement will execute a set of statements **only** if that condition is *true*. 
+1. Describe the meaning of *conditional statements*
+1. Use the `if` statement to conditionally execute a set of statements
+1. Use an `else` clause to provide an alternative set of statements
+1. Chain multiple *if-else* statements together
+
+## Conditional statements
+
+You already learned how to use the `Bool` type and various operators to evaluate logical expressions and compare values:
 
 ```swift
 var age = 19
 
+let isOfDrinkingAge = age > 21 // false, because 19 is not greater than 21
+```
+
+As was briefly mentioned in that lesson, those expressions become powerful using *conditional statements* which allow you to write pieces of code which can be executed (or not executed) based on certain conditions. These are called *control flow* constructs, because they allow you to control the execution flow of your program, allowing your program's output to react to conditions, instead of always providing the same output.
+
+## If statement
+
+The `if` statement makes the execution of a set of statements (also called a *block of code*) following it conditional based on a `Bool` value or expression. The given block of code will be executed *only if* the expression evaluates to `true`:
+
+```swift
+let age = 19
+
 if age < 21 {
-    print("You're not allowed to drink!")
+	print("You are not allowed to drink, yet.")
 }
 
-// prints "You're not allowed to drink!"
+// prints "You are not allowed to drink, yet."
 ```
 
-* Explain that an *if* statement can provide an alternative set of statements, known as an *else clause*, for situations when the *if* condition is *false*.
+You can type this block of code in a new Playground file and experiment with giving `age` different values. You will notice that the `print` statement will no longer be executed once you increase the value of `age` to 21 or higher. 
+
+Now try to extend the program by adding a second `if` statement which prints a message if the `age` is greater than or equal to 21. 
+
+You will have ended up with something like this:
 
 ```swift
-var myHeight = 50
+let age = 55
+
+if age < 21 {
+	print("You are not allowed to drink, yet.")
+}
+
+if age >= 21 {
+	print("You are old enough to drink. 🍻")
+}
 
-if myHeight > 60 {
-    print("You're free to go on the ride, enjoy!")
+// prints "You are old enough to drink. 🍻"
+```
+
+## Else clause
+
+This seems hard to read and will also lead to duplicated work should the condition ever change in the future. Thankfully, Swift provides a better way to provide an alternative block of code to an `if` statement, an *else clause*:
+
```

Comment:
Instead of summarizing this by stating "This seems hard to read", I would prefer that we go deeper into what that is by showing them another example.

```swift
let age = 69

if age < 50 {
    print("You are not middle aged.")
}

if !(age < 50) {
    print("You are at least half a century old!")
}

// prints "You are at least half a century old!"
```